whois-api/src/main/java/net/ripe/db/whois/api/rdap/WhoisRdapService.java

```diff
@@ -76,7 +78,8 @@ public WhoisRdapService(final RdapQueryHandler rdapQueryHandler,
                             final SourceContext sourceContext,
                             @Value("${rdap.public.baseUrl:}") final String baseUrl,
                             final RdapRequestValidator rdapRequestValidator,
-                            @Value("${rdap.search.max.results:100}") final int maxResultSize) {
+                            @Value("${rdap.search.max.results:100}") final int maxResultSize,
```

Comment:
These arguments are very similar, I suggest documenting what the difference between them is (i.e. in JavaDoc)